es-design-system/pages/molecules/es-video.vue

```diff
@@ -0,0 +1,51 @@
+<!-- eslint-disable vuejs-accessibility/no-static-element-interactions -->
+<template>
+    <div>
+        <h1>
+            Video
+        </h1>
+        <p>
+            Shows the embed video when the cover is clicked.
+        </p>
+        <div>
+            <es-video
+                :alt-text="altText"
+                class="my-450 mb-200 mb-lg-0"
+                :cover-image-url="placeholderEsThumbnail"
+                :embed-url="embedUrl" />
+        </div>
+        <ds-doc-source
+            :comp-code="compCode"
+            comp-source="es-vue-base/src/lib-components/EsVideo.vue"
+            :doc-code="docCode"
+            doc-source="es-design-system/pages/molecules/es-video.vue" />
+    </div>
+</template>
+<script>
+import placeholderEsThumbnail from '@/assets/placeholder/energysage_thumbnail.png';
```

Comment:
I have the video thumbnail icon currently stored under the assets/placeholder folder, but I am not sure where it should be stored?